themes/default/content/aws-summit/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Pulumi + AWS Summit London
+meta_desc: Join Pulumi at AWS Summit London to learn about the latest developments in infrastructure as code. Meet us at booth B14, April 24, 2024
+meta_image:
+type: page
+layout: aws-summit
+
+
+links:
+    items:
+        - heading: Request a Demo
+          description: See how Pulumi can help you ship infrastructure faster, and manage your AWS resources at scale. Ready for a change?
+          action: Talk with an Engineer
+          link: https://info.pulumi.com/aws-summit/
+
+workshops:
+    items:
+      - title: Getting Started with Infrastructure as Code on AWS
+        date: May 9, 2024
+        description: In this workshop, you will learn the fundamentals of Infrastructure as Code through a series of guided exercises using the Pulumi Cloud Engineering platform.
+        link: /resources/getting-started-with-iac-on-aws/
+        action: Register Now
+      - title: Advanced CI/CD for AWS using Pulumi and GitHub Actions
+        date: May 16, 2024
+        description: Learn advanced topics that make up a robust infrastructure CI/CD pipeline through guided exercises.
+        link: /resources/advanced-cicd-aws-pulumi-github-actions/
+        action: Register Now
+
+templates:
+    items:
+        - heading: Container Service Templates
+          description: Pulumi program templates are the fastest way to deploy container services on AWS.
+          image: /images/kubecon/templates-container-service-aws.png
```

Comment:
why don't you just link it to the origin file - /templates/container-service/aws/architecture.png